### PR TITLE
[ui] Feature/mainnet configs

### PR DESCRIPTION
**Pull request by 59023g**

### CODE REVIEW

File: ui/core/src/config.mainnet.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "sifAddrPrefix": "sif",
+  "sifApiUrl": "",
+  "sifWsUrl": "",
+  "sifRpcUrl": "",
+  "sifChainId": "",
```
Comment (utx0) [inline ui/core/src/config.mainnet.json:6]: sifChainId = `sifchain`

File: ui/core/src/config.mainnet.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "sifAddrPrefix": "sif",
+  "sifApiUrl": "",
+  "sifWsUrl": "",
+  "sifRpcUrl": "",
```
Comment (utx0) [inline ui/core/src/config.mainnet.json:5]: sifRpcUrl = `rpc.sifchain.finance`

File: ui/core/src/config.mainnet.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "sifAddrPrefix": "sif",
+  "sifApiUrl": "",
```
Comment (utx0) [inline ui/core/src/config.mainnet.json:3]: sifApiUrl = `api.sifchain.finance`

File: ui/core/src/config.mainnet.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  "sifWsUrl": "",
```
Comment (utx0) [inline ui/core/src/config.mainnet.json:4]: sifWsUrl = `ws.sifchain.finance`

**Review by brentmcd — DISMISSED** [review]

**Review by utx0 — DISMISSED** [review]

**Review by intl-man — DISMISSED** [review]

File: ui/core/src/config.ts [hunk 5]
```diff
@@ -60,6 +61,7 @@ export function getConfig(
   const allAssets = [...sifchainAssets, ...ethereumAssets].map(cacheAsset);
 
   const configMap: ConfigMap = {
+    mainnet: parseConfig(mainnnetconfig as ChainConfig, allAssets),
     localnet: parseConfig(localnetconfig as ChainConfig, allAssets),
     monkeybars: parseConfig(monkeybarsconfig as ChainConfig, allAssets),
```
Comment (utx0) [inline ui/core/src/config.ts:66]: You guys might want to change this key to be more generic ie `testnet` as we have dropped monkeybars and are now on merry-go-round, and this is likely to be dropped for a new network very soon.

Comment (59023g) [reply ui/core/src/config.ts]: Good call. Done

**Review by utx0 — APPROVED** [review]

**Pull request merged**